huisbot/Services/OsuApiService.cs

```diff
@@ -200,6 +200,8 @@ public async Task<bool> EnsureAccessTokenAsync()
     try
     {
       // Get the score from the API and check whether a 404 was returned. If so, the score was not found.
+      _http.DefaultRequestHeaders.Remove("x-api-version");
+      _http.DefaultRequestHeaders.Add("x-api-version", "20220705");
```

Comment:
This should be implemented in the http client builder in Program.cs instead.

Reply:
Ok, I'm on it =)

Reply:
Done.